pkgs/development/python-modules/servefile/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pyopenssl
+, pytestCheckHook
+, requests
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "servefile";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sebageek";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-/ZEMZIH/ImuZ2gh5bwB0FlaWnG/ELxfBGEJ2SuNSEb8=";
+  };
+
+  propagatedBuildInputs = [ pyopenssl ];
+
+  checkInputs = [ pytestCheckHook requests tox ];
+  # Skip a test which fails in the nix build environment.
+  disabledTests = [ "test_abort_download" ];
+  pythonImportsCheck = [ "servefile" ];
+
+  meta = with lib; {
+    description = "Serve files from shell via a small HTTP server";
+    homepage = "https://github.com/sebageek/servefile";
```

Comment:
Missing license

Reply:
I wasn't able to find a license on the project (https://github.com/sebageek/servefile/).

Reply:
The license is [`GPLv3+`](https://github.com/sebageek/servefile/blob/master/setup.py#L35).

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/147307/commits/56357747329500928809fb77566d69b0760c6a00.